traffic_portal/app/src/scripts/config.js

```diff
@@ -23,4 +23,4 @@
 
 angular.module('config', [])
 
-.constant('ENV', { api: { root:'/api/4.0/', legacy: '/api/1.5/', stable: "/api/3.1/" } });
```

Comment:
Why remove the `legacy` property? Can it not be incremented to API v2?

Reply:
It could, but it wouldn't matter because that's unused. It was added to allow TP to still show cache server configuration files after moving to APIv2 - since those routes no longer work we have no further use for "legacy" afaik.